### ISSUE TITLE
Negative amounts not aligned

### ISSUE THREAD

**Issue by markokocic** [simonmichael/hledger#1345]
Looks like negative amounts in reports are not anymore aligned with positive amounts. I am at the latest git version at the moment.

Steps to reproduce:
```
~/tmp $ cat test 
2020/09/01 I
  assets:a  1
  assets:a:d  1
  assets:c  1
  income

2020/09/01 A
  expenses:a  1
  assets:b

~/tmp $ hledger bs -f test 
Balance Sheet 2020-09-01

             || 2020-09-01 
=============++============
 Assets      ||            
-------------++------------
 assets:a    ||          1 
 assets:a:d  ||          1 
 assets:b    || -1 
 assets:c    ||          1 
-------------++------------
             ||          2 
=============++============
 Liabilities ||            
-------------++------------
-------------++------------
             ||            
=============++============
 Net:        ||          2 
~/tmp $ hledger bal -f test 
                   1  assets:a
                   1  assets:a:d
  -1  assets:b
                   1  assets:c
                   1  expenses:a
  -3  income
--------------------
                   0
~/tmp $ 

```

Previous and expected behaviour is that both negative and positive amounts are right aligned.

**Comment by simonmichael** [comment]
Yikes! Thank you, good catch.

**Comment by simonmichael** [comment]
@Xitian9, the new stripAnsi doesn't remove this ANSI:
```
ghci> stripAnsi $ showAmountWithoutPrice True (-1)
-- "\ESC[31m-1\ESC[m"
```
I don't quite see why not, do you ?

**Comment by simonmichael** [comment]
Fixed by 1.19.1.
